src/applications/bmqbrkr/etc/bmqbrkrcfg.json

```diff
@@ -1,6 +1,6 @@
 {
     "taskConfig": {
-        "allocatorType": "STACKTRACETEST",
```

Comment:
STACKTRACETEST allocator is for debug only, it is slow. For better user experience, we should provide COUNTING allocator as a default